tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py27, py3
+
+[testenv]
+commands =
+    python setup.py test -q
+deps =
+    .[test]
```

Comment:
This is better spelled `extras = test`.

Also, there's no trailing newline, you might want have a talk with your text editor.